autoconfigure/storage-logzio/src/main/java/zipkin/autoconfigure/storage/logzio/ZipkinLogzioStorageProperties.java

```diff
@@ -1,6 +1,6 @@
 package zipkin.autoconfigure.storage.logzio;
 
-import ch.qos.logback.classic.Logger;
```

Comment:
My guess is this was completely accidental, but it caused a hard dependency on logback, which zipkin-server doesn't use (it uses normal spring boot, but managed by log4j2). Reverting this is the simplest way to avoid a dependency problem.

Reply:
Nice catch!